Datagrid/ListMapper.php

```diff
@@ -125,7 +125,7 @@ public function add($name, $type = null, array $fieldDescriptionOptions = array(
         } elseif ($fieldDescription->getLabel() !== false) {
             $fieldDescription->setOption(
                 'label',
-                $this->admin->getLabelTranslatorStrategy()->getLabel($fieldDescription->getLabel(), 'list', 'label')
+                $fieldDescription->getLabel()
             );
         }
```

Comment:
This change does not make sense in this way.
You are trying to reset an already set value.

```php
$fieldDescription->getLabel() === $fieldDescription->getOption('label');
```

https://github.com/sonata-project/SonataAdminBundle/blob/3.x/Admin/BaseFieldDescription.php#L467